### ISSUE TITLE
updates for fw v1.4/1.5 ?

### ISSUE THREAD

**Issue by ghost** [smealum/udsploit#17]
title says it!, for rpghax vr homebrew..
should be all that was for v11.3

https://github.com/TiniVi/safehax/issues/36

on list https://3ds.guide/homebrew-launcher-(soundhax).html , est

**Comment by paulovnas** [comment]
I'm waiting for that, too.

**Comment by ihaveamac** [comment]
but the exploit was fixed in 11.4

**Comment by ghost** [comment]
^ don't know, rpghax only works better with homebrew 1.1.0 , issues in crashing on exit of files with other versions, could be down 2 fw 11.4/11.5 maybe.., but I only had 11.5 to test so son't know on fw 11.4, the rphhax exploit was little buggy if wish to load are not at random, but as for these programs don't work fully as intended.

**Comment by ihaveamac** [comment]
no, udsploit is an exploit that was fixed in 11.4. so it cannot and will not be updated for 11.4+......

**Comment by ghost** [comment]
Sorry I see ^, thought smealum was the maker .

anyways don't matter and thanks.

**Comment by chronoss09** [comment]
https://github.com/smealum/udsploit/issues/16

**Comment by chronoss09** [comment]
> no, udsploit is an exploit that was fixed in 11.4. so it cannot and will not be updated for 11.4+......

not fixed yet
